src/atoms/NumberedListItem.js

```diff
@@ -22,4 +23,9 @@ const RecipeStep = styled.div`
 	}
 `;
 
-export { RecipeStep };
+const NumberedListItem = ({ children }) => (
+	<NumberedListItemWrapper>
+		<span>{children}</span>
```

Comment:
What does the span do?

Reply:
I'm not sure why, but somehow it doesn't work without it :c

Reply:
And this is why we need both NumberedListItemWrapper and NumberedListItem

Reply:
You can merge this as it is, but please also create an issue about the span, so that I can take a closer look some late night when I have nothing else to do